e2e/ctl_v3_lease_test.go

```diff
@@ -138,21 +217,28 @@ func leaseTestKeepAliveOnce(cx ctlCtx) {
 	}
 }
 
-func leaseTestRevoke(cx ctlCtx) {
+func leaseTestRevoked(cx ctlCtx) {
```

Comment:
Why we have `leaseTestRevoked`? Just keep `leaseTestRevoke`?

Reply:
the same reasoning as `leaseTestGrantLeaseListed` will resolve if it does not make sense.

Reply:
I see. How about changing `leaseTestTimeToLiveExpire` to just error out, without returning error, and revert other changes?

Reply:
I was only worried about handling the proper error content when used by other tests but this is probably getting ahead of myself as the nested nature will get kinda ugly. Will resolve.

Reply:
If I don't return error in `leaseTestTimeToLiveExpire` how can I handle fatal below? I see this a lot in the tests where functions are reused, just making sure I understand your intent.

https://github.com/coreos/etcd/blob/22127895d818b161e7e96343d6b2745d9e23d879/e2e/ctl_v3_auth_test.go#L752-L764

Reply:
> If I don't return error in leaseTestTimeToLiveExpire how can I handle fatal below?

I don't understand.

The reason why we don't return an error, is that we want to know which line it's fatal-ing at.
If we return errors, the errors should be descriptive, as this PR does.

Either way works. Just want to make it consistent.

For example, `leaseTestRevoke` in this PR is mixing error returning and `cx.t.Errorf`.

Reply:
understood, thanks